chains/v20/preConfigured/detailsDev/713daf193a6301583ff467be736da27ef0a72711b248927ba413f573d2b38e44.json

```diff
@@ -31,9 +31,5 @@
             "name": "Parity"
         }
     ],
-    "options": [
```

Comment:
that options shouldn't be deleted

Reply:
yes, that's why I don't emerge it still: the matter is that flow delete all files by CI and then fill it again with script, so we need to skip them before CI